Components/Modal/modal.css

```diff
@@ -0,0 +1,54 @@
+.modal-container{
+    position: relative;
+    margin-top:15rem;
+}
+
+
+.modal {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    max-width: 100%;
+  
+    background-color: var(--primary-light-color);
+    padding: 4rem;
+    border-radius: 5px;
+    box-shadow: 0 3rem 5rem rgba(0, 0, 0, 0.3);
+    z-index: 12;
+  }
+
+
+  .close-modal {
+    position: absolute;
+    top: 0;
+    right:0;
+    font-size: 5rem;
+    color: var( --primary-text-color);
+    cursor: pointer;
+    border: none;
+    background: none;
+  }
+
+  .modal-btns{
+      display: flex;
+      justify-content:space-around;
+      margin-top:3rem;
```

Comment:
In line number 16 you have used `border-radius: 5px;` and here you have used `margin-top:3rem;` . It would be better if the values are consistent, either rems for both or px for both.

Reply:
okay, will work on the changes.Thanks for the review @amantulsyan35